lib/bundler_api/gem_info.rb

```diff
@@ -46,10 +48,10 @@ def deps_for(gems = [])
 SQL
       end
 
-    deps = {}
+    deps = {} # this needs to be an ordered hash
```

Comment:
Yes, I put that note there because I would like to do some stuff that is done after the query inside the query. Didn't had the time yet